syntax/apiblueprint.vim

```diff
@@ -20,6 +22,8 @@ syntax region apibActionRelation start=/^[-+*] Relation: .*$/ end=/$/ oneline co
 syntax match apibHeadersSectionKey /^[^:]*\ze:/ contained
 syntax match apibHeadersSectionValue /:.*/ contained
 
+syntax region apibJSON start=/^\s*{/ end=// contained containedin=markdownCodeBlock contains=@json
```

Comment:
Isn't that just a matter of having `[{\{]` instead of `{`? Anyway, by the time you responded, I stopped working with API blueprints and I don't think I will in near future so I have little motivation to work on this any further. :-(

Regarding checking for `end`, I vaguely remember that if the JSON block had missing or superflouos closing braces, the colors were off, which was good enough for my editing needs.